.nextstrain/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
This directory shouldn't exist (see my other comment), so this gitignore should go away.